cpu/esp_common/esp-wifi/esp_wifi_netdev.c

```diff
@@ -903,6 +903,25 @@ static wifi_config_t wifi_config_ap = {
 };
 #endif /* (defined(MCU_ESP8266) && !defined(MODULE_ESP_NOW)) || defined(MODULE_ESP_WIFI_AP) */
 
+
+#if (defined(MCU_ESP32) && defined(MODULE_ESP_WIFI_CSI))
+void __attribute__((weak)) esp_wifi_csi_rx_cb(void *ctx, wifi_csi_info_t *data)
+{
+    (void)ctx;
+    printf("WiFi CSI %p\n", data);
```

Comment:
Shouldn't the default callback function print some information instead of the pointer to the channel state information?